sdk/core/azure-servicemanagement-legacy/HISTORY.rst

```diff
@@ -3,7 +3,7 @@
 Release History
 ===============
 
-0.20.7 (2018-XX-XX)
+0.20.7
```

Comment:
Add XX-XX corrupts the tag for some reason.

Reply:
Please add Unreleased or something to make it clear that it's not released. Praveen might know the right syntax.

Reply:
I can add that right now. But I discussed again with @weshaggard and we plan to eventually switch all the CHANGELOG.rst to CHANGELOG.md. I will start another issue to track that. When that happens it will be nice to stick with the changelog convention currently being used for all the other repos which is to keep the latest unreleased version (without date, or any other marker) at the top.

Reply:
That's not the convention I disagree, discussed with @weshaggard already that unreleased deserved to be marked explicitly unreleased

Reply:
@weshaggard @scbedd @AlexGhiondea @ramya-rao-a what do you think. Should we update the changelog guidance so we mark the latest unreleased version as unreleased.

i.e
# [Version Number] Unreleased

Reply:
Lets take that discussion to https://github.com/Azure/azure-sdk/pull/852. However I'm fine having a `<version> Unreleased` convention.

Also for reference here is the teams thread where @lmazuel and I agreed on this https://teams.microsoft.com/l/message/19:b97d98e6d22c41e0970a1150b484d935@thread.skype/1572639606659?tenantId=72f988bf-86f1-41af-91ab-2d7cd011db47&groupId=3e17dcb0-4257-4a30-b843-77f47f1d4121&parentMessageId=1572639606659&teamName=Azure%20SDK&channelName=Language%20-%20Python&createdTime=1572639606659.